src/seedu/addressbook/data/person/Block.java

```diff
@@ -0,0 +1,18 @@
+package seedu.addressbook.data.person;
+
+/**
+ * 
+ * @author joellim
+ *  The block part of an address
```

Comment:
comments should be in active verb form.
`Represents the block information in an address` may be more appropriate